repubblica-digitale/main.py

```diff
@@ -45,7 +45,7 @@ def main(self):
         scopes = ["https://www.googleapis.com/auth/spreadsheets"]
 
         spreadsheet_id = '1BV95DWqrytq1kSyP2EdFkJ0PSuGM5gdx0jhuS144tQI'
-        range_name = 'Grafici!A1:C200'
+        range_name = 'Grafici!A1:C400'
```

Comment:
:cold_sweat: 

Can't we use `'Grafici!A:C'`?

Reply:
yeah this is such awful, I found out [this](https://developers.google.com/sheets/api/guides/concepts#a1_notation) right now. So, yes, we can, hoping that it will not add too much overhead